cms/envs/production.py

```diff
@@ -603,3 +603,5 @@ def get_env_setting(setting):
 
 ################# show account activate cta after register ########################
 SHOW_ACCOUNT_ACTIVATION_CTA = ENV_TOKENS.get('SHOW_ACCOUNT_ACTIVATION_CTA', SHOW_ACCOUNT_ACTIVATION_CTA)
+
+LANGUAGE_COOKIE_NAME = ENV_TOKENS.get('LANGUAGE_COOKIE')
```

Comment:
I am planning to go through this route to avoid any sync issues.

- First merge this PR with this line
- Create PR in internals with both settings name `LANGUAGE_COOKIE_NAME` and `LANGUAGE_COOKIE` 
- Create PR in platform to drop this line from both lms and cms (`LANGUAGE_COOKIE_NAME = ENV_TOKENS.get('LANGUAGE_COOKIE')`)
- Drop `LANGUAGE_COOKIE` from internals

Reply:
Sounds good!

As an alternative, I think you could change this line to read both settings (e.g. `ENV_TOKENS.get('LANGUAGE_COOKIE', None) or ENV_TOKENS.get('LANGUAGE_COOKIE_NAME')`) and then make one fewer PR overall -- but either way seems fine.

Reply:
@jmbowman can you please guide me, which domain should be appropriate for this cookie in this middleware?